packages/dds/tree/src/shared-tree-core/branch.ts

```diff
@@ -141,16 +165,28 @@ export class SharedTreeBranch<TEditor extends ChangeFamilyEditor, TChange> exten
 	private popTransaction(): [
 		GraphCommit<TChange>,
 		GraphCommit<TChange>[],
+		RevisionTag,
 		RepairDataStore | undefined,
 	] {
-		const { startRevision, repairStore } = this.transactions.pop();
+		const { startRevision, revision, repairStore } = this.transactions.pop();
 		const commits: GraphCommit<TChange>[] = [];
 		const startCommit = findAncestor([this.head, commits], (c) => c.revision === startRevision);
 		assert(
 			startCommit !== undefined,
 			0x593 /* Expected branch to be ahead of transaction start revision */,
 		);
-		return [startCommit, commits, repairStore];
+		return [startCommit, commits, revision, repairStore];
+	}
+
+	/**
+	 * TODO: Return something, prob undo result
```

Comment:
As mentioned below, I don't think we need to retun anything, but we do want this to be documented. Including the fact that undo within a transaction is not yet supported.

Reply:
My bad, we do need to return a boolean to say whether something was undone (since the undo stack may be empty).